pkg/service/generateConfig/generateConfig.go

```diff
@@ -46,19 +46,12 @@ test:
   networkName: ""
   # example: "test-set-1": ["test-1", "test-2", "test-3"]
   tests: 
-  globalNoise: |-
-    {
-      "global": {
-        "body": {},
-        "header": {}
-      },
-      "test-sets": {
-        "test-set-name": {
-          "body": {},
-          "header": {}
-        }
-      }
-    }
+  # to use globalNoise, please follow the guide at the end of this file.
+  globalNoise:
+    global:
+      body: {}
+      header: {}
+    test-sets:
```

Comment:
may be just remove this test-sets param from here, people can understand how to use it from comments below.

Reply:
done !